app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/triage/multipledisposals/how_many_properties.scala.html

```diff
@@ -21,68 +21,90 @@
+)
+
+@(
+  form: Form[Int],
+  backLink: Call,
+  hasCreatedDraftReturn: Boolean,
+  isAmend: Boolean
+)(
+  implicit request: RequestWithSessionData[_],
+  messages: Messages,
+  appConfig: ViewConfig
 )
 
-@(form: Form[Int], backLink: Call, hasCreatedDraftReturn: Boolean, isAmend: Boolean)(implicit request: RequestWithSessionData[_], messages: Messages, appConfig: ViewConfig)
 @key = @{"multipleDisposalsNumberOfProperties"}
-@title = @{messages(s"$key.title")}
-@hasErrors = @{form.hasErrors || form.hasGlobalErrors}
 
-@mainTemplate(title = title, hasErrors = hasErrors, userType = request.userType) {
+@title = @{messages(s"$key.title")}
 
-    @backLinkComponent(backLink)
+@hasErrors = @{form.hasErrors || form.hasGlobalErrors}
 
-    @if(hasErrors) {
-        @errorSummary(form)
-    }
+@detailsContent = {
+  <p class="govuk-body">@{messages(s"$key.details.p1")}</p>
+  <p class="govuk-body">@{messages(s"$key.details.p2")}</p>
+  <p class="govuk-body">@{Html(messages(s"$key.details.link", appConfig.tranferringOwnershipHelp))}</p>
+}
 
-    @caption(messages("triage.caption"))
+@layout(
+  title,
+  hasErrors = hasErrors,
+  backLinkUrl = Some(backLink.url)
+) {
+  @if(hasErrors) {
+    @errorSummary(form)
+  }
 
-    @formWrapper(triage.routes.MultipleDisposalsTriageController.howManyDisposalsSubmit()) {
+  @formWrapper(triage.routes.MultipleDisposalsTriageController.howManyDisposalsSubmit()) {
+    <span class="govuk-caption-xl">
+      @messages("triage.caption")
+    </span>
 
-        @pageHeading(title)
+    <h1 class="govuk-heading-xl">
+      @title
+    </h1>
 
-        <p>@messages(s"$key.p1")</p>
-        <p>@messages(s"$key.p2")</p>
+    <p class="govuk-body">
+      @messages(s"$key.p1")
+    </p>
+    
+    <p class="govuk-body">
+      @messages(s"$key.p2")
+    </p>
 
-        @textInput(
-            fieldId = key,
-            label = Html(title),
-            helpText = None,
-            errorKey = form.error(key).map(e => e.message),
-            value = form.data.getOrElse(key, ""),
-            hasErrors = hasErrors,
-            autoComplete = AutoCompleteType.On,
-            inputClass = Some("input--xsmall"),
-            labelClass = Some("visually-hidden")
-        )
-        @details(
-            messages(s"$key.details.summary"),
-            Html(
-            s"""
-            | <p>${messages(s"$key.details.p1")}</p>
-            | <p>${messages(s"$key.details.p2")}</p>
-            | <p>${messages(s"$key.details.link", appConfig.tranferringOwnershipHelp)}</p>
-            |""".stripMargin
-            )
-        )
+    @govukInput(Input(
+      id = key,
+      name = key,
+      inputmode = Some("numeric"),
+      value = Some(form.data.getOrElse(key, "")),
+      errorMessage = form.error(key).map(e => ErrorMessage(
+        content = Text(messages(s"$key.${e.message}")),
+        visuallyHiddenText = Some(messages("generic.error"))
+      )),
+      classes = "govuk-input--width-3"
+    ))
```

Comment:
we have no label defined here - this field is not accessible like this, even if the label is visually hidden, as it was before, it still needs to exist for screenreaders and tests to read

Reply:
an example of a visually hidden label:
```
      label = Label(
        content = Text(title),
        classes = "govuk-visually-hidden"
      ),
```

Reply:
Done